samples/excel/tables/format-table.yaml

```diff
@@ -0,0 +1,194 @@
+id: ''
+gist: ''
+source: EXCEL
+author: Microsoft
+name: Format table
+description: Format a table
+script:
+  content: |+
+    $('#create-table').click(createTable);
+    $('#format-table').click(formatTable);
+
+    /** Create a new table with sample data */
```

Comment:
Comment doesn't say much more than what the function name. I suggest avoiding this as it clutters the code.